inc/telemetry.class.php

```diff
@@ -299,6 +299,11 @@ static public function cronTelemetry($task) {
     * @return string
     */
    private static final function getUuid($type) {
+      global $DB;
+
+      if ($DB === null) {
+         return Toolbox::getRandomString(40);
```

Comment:
I think it should just return null in this case. If null value is a problem for #4091 changes, then maybe the css function should be modified for `$instance = md5((Telemetry::getInstanceUuid() || Toolbox::getRandomString(40)). GLPI_VERSION);`.

Reply:
Problem is `Telemetry::getInstanceUuid()` fails n this case, but not in a way we expect.

Reply:
I agree that it should not fail. But why returning a random string when there is no uuid ? I think this function should not return a volatile value, it should just return null if UUID cannot be retrieved or stored after generation.
If you return null here, then the `Html::css` function will almost act as it was before #4091 changes in the install process.
By the way, since #4091, version param used for plugin CSS files is based on GLPI version and not on plugin version.

Reply:
I agree, method itself should return null; and this would act just as before 4091; but using md5 of the version or the raw version does not change really the "problem" that has been addressed in the PR.
You're right for the version, this is not the correct one that is used right now...

Reply:
I really do not know why the telemetry method (in fact, this fails in Config) fails kind of silently, that's why I've changed here.
I do not want to spend too much time to fix that; I was not convinced by this full "version issue" stuff.